audb/core/publish.py

```diff
@@ -171,60 +171,81 @@ def _find_media(
     num_workers: int,
     verbose: bool,
 ) -> set[str]:
-    r"""Find archives with new, altered or removed media and update 'deps'."""
+    """Find archives with new, altered or removed media and update 'deps'.
+
+    The function alters the dependency table entries of ``deps`` in place
```

Comment:
Reading the docstring I think that one could possibly improve its readibility by rephrasing the tasks that are done by the function such that it uses a bulleted list:
For example:

```

The function alters the dependency table entries of ``deps`` in place
by:

* adding new entries for new media files
* removing entries for removed media files
* updating entries for altered media files

It further collects all media archives,
```

Reply:
Good suggestion, I changed the docstring to:

```
    The function alters the dependency table entries of ``deps`` in place
    by

    * adding entries for new media files
    * removing entries for removed media files
    * updating entries for altered media files

    It further collects all media archives,
    that are affected by those media files.
```

Reply:
For me it looks better, therefore resolving conversation.